.github/workflows/release.yml

```
@@ -0,0 +1,18 @@
+name: Create GitHub Release
+
+on:
+  # Run on demand
+  workflow_dispatch:
+
+  # Run on push to main when .bumpversion.cfg version is updated
+  push:
+    branches:
+      - main
+    paths:
+      - .bumpversion.cfg
+
+jobs:
+  release:
+    uses: plus3it/actions-workflows/.github/workflows/release.yml@41056300ba78d8be98b45f057aea246f8c4f30cf
+    secrets:
+      release-token: ${{ secrets.GH_RELEASES_TOKEN }}
```

Let's disable the mockstack test on the release workflow...

```suggestion
  release:
    uses: plus3it/actions-workflows/.github/workflows/release.yml@41056300ba78d8be98b45f057aea246f8c4f30cf
    with:
      mockstacktest-enable: false
    secrets:
      release-token: ${{ secrets.GH_RELEASES_TOKEN }}
```

---

mockstack is still failing; rather than "skipping" the job

---

Is this resolved, the conversation can be resolved if so.